app/EnrolmentLesson.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App;
+use App\Enrolment;
+use App\Lesson;
+use Illuminate\Database\Eloquent\Model;
+
+class EnrolmentLesson extends Model
```

Comment:
This is a pivot table right? 

I think you can extend the `Pivot` model? 

You can also drop the trailing `s` in the table name and Laravel will guess the table name correctly.